string node
---
default_cfg_fkie/Description[] items